README.rst

```diff
@@ -1,61 +1,50 @@
-django-spanner
-==============
+Cloud Spanner support for Django
+================================
 
-ORM plugin for using Cloud Spanner as a database for Django.
+ORM plugin for using Cloud Spanner as a `database backend
+<https://docs.djangoproject.com/en/2.2/ref/databases/#using-a-3rd-party-database-backend>`__
+for Django.
 
-🚨THIS CODE IS STILL UNDER DEVELOPMENT🚨
-========================================
 
-Table of contents
------------------
-
--  `Installing it <#installing-it>`__
--  `Using it <#using-it>`__
-
-   -  `Format <#format>`__
-   -  `Example <#example>`__
+Installation
+------------
 
--  `Functional tests <#functional-tests>`__
--  `Django integration tests <#django-integration-tests>`__
+To use this library you'll need a Google Cloud Platform project with the Cloud
+Spanner API enabled. See the `Cloud Spanner Python client docs
+<https://github.com/googleapis/python-spanner/#quick-start>`__ for details.
 
-   -  `django\_test\_suite.sh <#django_test_suitesh>`__
+Use the version of ``python-spanner-django`` that corresponds to your version
+of Django.  For example, ``python-spanner-django`` 2.2.x works with Django
+2.2.y. (This is the only supported version at this time.)
 
-      -  `Environment variables <#environment-variables>`__
-      -  `Example run <#example-run>`__
+The minor release number of Django doesn't correspond to the minor release
+number of ``python-spanner-django`` Use the latest minor release of each.
 
-   -  `Parallelization script <#parallelization-script>`__
+To install from PyPI:
 
-      -  `Environment variables <#environment-variables>`__
-      -  `Example run <#example-run>`__
+.. code:: shell
 
--  `Limitations <#limitations>`__
--  `How it works <#how-it-works>`__
+    pip3 install django-google-spanner
 
-   -  `Overall design <#overall-design>`__
-   -  `Internals <#internals>`__
 
-Installing it
--------------
+To install from source:
 
-Use the version of django-spanner that corresponds to your version of
-Django. For example, django-spanner 2.2.x works with Django 2.2.y. (This
-is the only supported version at this time.)
+.. code:: shell
 
-The minor release number of Django doesn't correspond to the minor
-release number of django-spanner. Use the latest minor release of each.
+    git clone git@github.com:googleapis/python-spanner-django.git
+    cd python-spanner-django
+    pip3 install -e .
 
-.. code:: shell
 
-    pip3 install --user .
+Use
```

Comment:
[nit] Instead of `Installing it` and `Using it` gerunds as section names, I'd rather see nouns like `Installation` and `Usage`, as more semantically aligned with `Limitations`.

Reply:
Agreed, I changed "use" and the last instance of "installing it".